my-app/src/services/LogInService.js

```diff
@@ -0,0 +1,24 @@
+import api from '../config/api';
+import { actions } from '../redux/login/actions';
+
+function setCredentialsInLocalStorage(response) {
+  localStorage.setItem('authorization', response.data.token);
+}
+
+function getAuthenticatedLogIn(values, dispatch) {
+  const queryParams = { user: values.mail, password: values.password };
+  api.get('/users', queryParams).then(response => {
```

Comment:
This service should only contain `API` calls, otherwise this file will be very coupled to `redux/thunk` implementation